src/ir.h

```diff
@@ -130,6 +128,22 @@ struct Block {
   Expr* first;
 };
 
+struct Catch {
+  WABT_DISALLOW_COPY_AND_ASSIGN(Catch);
+  Catch() = delete;
+  explicit Catch(Expr* first);
+  Catch(Var var, Expr* first);
+  ~Catch();
+  Location loc;
+  Var var;
+  struct Expr* first;
+  bool IsCatchAll() const {
```

Comment:
What's the benefit of distinguishing catch vs. catch_all here rather than in the try block?